#goal
string ftp_server
string username
string password
string remote_path
string remote_folder
string local_path
---
#result
bool success
---
#feedback
string current_folder
